EventStore.API/Model/EventStore/SchedulingEventType.cs

```diff
@@ -0,0 +1,10 @@
+namespace EventStore.API.Model.EventStore
+{
+    public enum SchedulingEventType
+    {
+        STEP_1,
+        STEP_2,
+        STEP_3,
```

Comment:
Can you rename these step types? For example doctor chosen, date chosen, idk what that step represents.

Reply:
Actually I can, but it's not a simple change, I would have to refactor business logic, vue.js code and fill out database data once again and it would require time that we do not have at the moment.